tests/conftest.py

```diff
@@ -135,7 +135,7 @@ def test_repository(transactional_db):
         repository_group_id=1,
         description="",
         performance_alerts_enabled=True,
-        tc_root_url="https://tc.example.com",
```

Comment:
This was causing a CSP error when running a few selenium tests since this example domain is not whitelisted.